src/coreclr/tools/superpmi/superpmi/superpmi.cpp

```diff
@@ -446,8 +443,8 @@ int __cdecl main(int argc, char* argv[])
                 }
             }
 
-            // Create a new CompileResult for this compilation (the CompileResult from the stored file is
-            // in originalCR if necessary).
+            delete mc->cr;
```

Comment:
```suggestion
            mc->Reset();
```
(also news up a CR, so below isn't needed)

Reply:
Makes sense, made that change. I also moved the call to the end of the loop and only when there is going to be a next iteration of the `repeatCount` loop.